nixos/modules/services/admin/docuum.nix

```
@@ -0,0 +1,37 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+
+let
+  cfg = config.services.docuum;
+in
+{
+  options.services.docuum = {
+    enable = mkEnableOption "docuum daemon";
+
+    threshold = mkOption {
+      description = "Threshold for deletion";
+      type = types.str;
+      default = "10 GB";
+    };
+  };
+
+  config = mkIf (cfg.enable) {
```

```
```suggestion
  config = mkIf cfg.enable {
```

These are redundant.